corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -975,21 +975,6 @@ hqDefine("cloudcare/js/form_entry/entries", function () {
     VideoEntry.prototype = Object.create(FileEntry.prototype);
     VideoEntry.prototype.constructor = FileEntry;
 
-    /**
-     * Represents a signature, which requires the user to upload a signature file.
-     */
-    function SignatureEntry(question, options) {
```

Comment:
what happens if a form contains a signature question? Will it just not who up on the UI?

Reply:
my understanding is that signature questions won't be an option initially and will be re-implemented later when drawing functionality is completed. 
@shubham1g5 @orangejenny please correct me if I'm missing something here.

Reply:
yeah, that's the intent

Reply:
I think the behaviour on this should remain same as what's there on production today aka we just show a message "Sorry, web entry cannot support this type of question"

Reply:
You get that message when the feature flag isn't on, correct? So for this change I should leave `SignatureEntry` as it is and just add the additional accepted file types to `ImageEntry`?

Reply:
I see, thanks for the clarification! @orangejenny